arch/arch_test.go

```diff
@@ -42,7 +42,7 @@ func exampleInfo() *nfpm.Info {
 			Contents: []*files.Content{
 				{
 					Source:      "../testdata/fake",
-					Destination: "/usr/local/bin/fake",
+					Destination: "/usr/share/bin/fake",
```

Comment:
@caarlos0 Was this meant to be `/usr/bin/fake` ?

Reply:
yes, good catch, fixed in 475f856